Form/Type/CollectionType.php

```diff
@@ -31,6 +34,30 @@ public function getParent()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['btn_add'] = $options['btn_add'];
+        $view->vars['btn_delete'] = $options['btn_delete'];
+        $view->vars['btn_catalogue'] = $options['btn_catalogue'];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
```

Comment:
Please prevent it from deprecation error.

Do something like this class: https://github.com/sonata-project/SonataAdminBundle/blob/master/Form/Type/ModelType.php#L65-L78